includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1619,6 +1643,13 @@ public static function filter_tag_row_actions( $actions, WP_Term $tag ) {
 				esc_attr__( 'Toggle error details', 'amp' ),
 				esc_html__( 'Details', 'amp' )
 			);
+
+			$actions['copy'] = sprintf(
+				'<button type="button" aria-label="%s" class="single-url-detail-copy button-link" data-error-json="%s">%s</button>',
+				esc_attr__( 'Copy to clipboard', 'amp' ),
+				esc_attr( self::get_error_details_json( $term ) ),
+				esc_html__( 'Copy to clipboard', 'amp' )
```

Comment:
Is the `aria-label` actually needed here?

```suggestion
				'<button type="button" class="single-url-detail-copy button-link" data-error-json="%s">%s</button>',
				esc_attr( self::get_error_details_json( $term ) ),
				esc_html__( 'Copy to clipboard', 'amp' )
```

Reply:
I was blindly following the other buttons on the screen, but you're right.